lib/common/tests/options/Makefile.am

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2022 the Pacemaker project contributors
+#
+# The version control history for this file may have further details.
+#
+# This source code is licensed under the GNU General Public License version 2
+# or later (GPLv2+) WITHOUT ANY WARRANTY.
+#
+
+AM_CPPFLAGS = -I$(top_srcdir)/include		\
+	      -I$(top_builddir)/include		\
+	      -I$(top_srcdir)/lib/common
+LDADD = $(top_builddir)/lib/common/libcrmcommon_test.la
```

Comment:
Once #2798 gets merged, you'll want to add `AM_CFLAGS = -DPCMK__UNIT_TESTING` right here - see [54c0620](https://github.com/ClusterLabs/pacemaker/pull/2798/commits/54c062060f120db94bbe25b2dc3d10bb6af40302).  You'll also want to add `-lcmocka` to `LDADD` - see [4dcd34a](https://github.com/ClusterLabs/pacemaker/pull/2798/commits/4dcd34a05b2bf8a7c6608167d7e064529f4e02a9).

Also, I feel like these Makefile.am files can probably get reduced a lot more.  I think these blocks at the top are just about identical at this point.